app/grandchallenge/archives/tasks.py

```diff
@@ -35,6 +38,34 @@ def add_images_to_archive(*, upload_session_pk, archive_pk, interface_pk=None):
         item.values.set([civ])
 
 
+@shared_task
```

Comment:
I would prefer if this was on a different queue as using the main one too much is causing problems like we saw over the holidays. Probably the acks-late is best, I think all that would be needed is to wrap this in an atomic transaction as there are no other side effects.

Reply:
Whilst we're here we can do the same with `update_archive_item_values` to improve the existing codebase :-)